fetcher.go

```diff
@@ -295,12 +323,16 @@ func updateSuccessServerTimingMetrics(timingHeaders []string, resourceType strin
 						}
 						networkLatencyMs := ttfbMs - m.Duration.Milliseconds()
 						fetchNetworkLatencyPeerSuccessMetric.WithLabelValues(resourceType).Observe(float64(networkLatencyMs))
+						latencyMillis = float64(networkLatencyMs)
```

Comment:
can we have a separate latency tracked metric for just responses that have server cache header set?

Reply:
@willscott Sure but should not matter here as we are measuring network latency sans the time it takes an L1 to process the request.

Reply:
But yeah no harm in putting it.